src/main/java/com/github/mkepDev/dependencyTrackBomPublish/DependencyTrackPublishBomTask.java

```diff
@@ -149,7 +150,9 @@ private void publishData(String host, String realm, String apiKey, String projec
             realm = realm.substring(0, host.length() - 1);
         }
 
-        String dtrackUrl = "http://" + host.replaceAll("http%://", "") + "/" + realm;
+        logger.info("Use new reqex");
+
+        String dtrackUrl = "http://" + host.replaceAll("https?://", "") + "/" + realm;
```

Comment:
Hi, I found your project really useful and I would like to use your plugin with https protocol for security reasons. Is there any specific reason why you replace https with http here?

Reply:
Hi

to be honest: I was just lazy.
The Docker image of dependency-track that I use, relies on a self signed-cert and there was no need to use https in my local network.

You can create a PR or wait for a few days and I will adjust this.